<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tree</class>
 <widget class="QWidget" name="Tree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="boxActions"/>
   </item>
   <item>
    <widget class="QWidget" name="verticalLayout_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLineEdit" name="editSearch">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearch">
          <property name="text">
           <string>Search</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/search.svg</normaloff>:/icons/search.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="listWidgetBox"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="boxUpload"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
